.github/workflows/metric.yml

```diff
@@ -0,0 +1,71 @@
+name: Metric and Commit Results
+
+on:
+  workflow_dispatch: # Allows manual triggering of the workflow
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+
+jobs:
+  compile-time:
+    name: Compile Time
+    runs-on: ubuntu-latest
+    # if: ${{ github.ref_name == 'main' }}
+    outputs:
+      time: ${{ steps.run.outputs.COMPILE_TIME }}
+    steps:
+      - uses: taiki-e/checkout-action@v1
+      - name: Checkout repo
+        uses: actions/checkout@v4
```

Comment:
checkout twice?

Reply:
https://github.com/oxc-project/monitor-oxc/blob/a0f34edc0c745d0a68ee4991e15046d398468548/.github/workflows/ci.yml#L102-L104, just copy your code, :smile: , I will fixed that